src/app/shared/menu/date/standard-date-range-dropdown.component.ts

```diff
@@ -32,8 +25,11 @@ import * as dateUtil from '../../util/date';
 export class StandardDateRangeDropdownComponent {
   @Input() standardRange: string;
   @Input() interval: IntervalModel;
-  @Output() custom = new EventEmitter();
   open = false;
+  @HostListener('window: scroll', [])
+  onWindowScroll() {
+    this.open = false;
```

Comment:
Would it be good to also do this on the other dropdowns?

Reply:
Ah, i see it's on the canned interval dropdown.  But maybe on the custom one too?

Reply:
I did actually have it on the custom date range dropdown but I took it out. I found that at my screen resolution, the window needed to be almost as tall as the viewport to see the Apply button. So if the user tried to scroll to it, the dropdown closes, and everyone is sad. I added these because it was gross to me that the dropdowns were fixed on scroll, but not being able to use the is kind of worse,